### PR TITLE
Jira docs 3.0 channel subscriptions edits

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

I'm adding some edits here for the PR at https://github.com/mattermost/mattermost-plugin-jira/pull/739. I'm making a separate PR, as there are a decent amount of changes I want to be reviewed in isolation.

Some things left to be done:

- [ ] Make sure all the links in the docs work correctly. @justinegeffen is there a way we can test this?
- [x] Add a picture of the channel subscriptions modal
- [x] Change the output of `/jira webhook` to point to docs instead of going in detail with Legacy Webhooks in output
- [ ] Clean up images folder in a separate PR. Right now the images are named `image.png`, `image (1).png`, and so on.

#### Ticket Link

Fixes https://mattermost.atlassian.net/browse/MM-31410

### CODE REVIEW

**Review by jfrerich — APPROVED** [review]
Looks good. I have a couple of non-blocking grammar suggestions.

**Comment by mickmister** [comment]
@hanzei Any ideas why ci lint is failing on `go.mod` issues in this docs PR?

**Comment by hanzei** [comment]
@mickmister Since the latest update of the plugin-ci orb `go mod tidy` is now enforced. The CI issue was fixed on `master` with https://github.com/mattermost/mattermost-plugin-jira/pull/746. Cherry-picking https://github.com/mattermost/mattermost-plugin-jira/commit/4f416fcb0480d24db45f46ba181bf1ad42853465 into this branch will fix CI.

**Comment by mickmister** [comment]
Self-reminder to changes references to the webhook URLs to use the new format that includes the instance ID, as pointed out here https://github.com/mattermost/mattermost-plugin-jira/pull/739#discussion_r602690177

File: docs/administrator-guide/frequently-asked-questions-faq.md [hunk 1]
```diff
@@ -40,39 +40,14 @@ The `<your-mattermost-url>`, `<your-port>`, `<your-secret>`, `<your-team-url>`,
 
 ## How do I handle credential rotation for the Jira webhook?
 
-You can generate a new secret in **System Console > Plugins > Jira**, and paste the new webhook URL in your Jira webhook configuration. This might result in downtime of the Jira plugin, but it should only be a few minutes at most.
+You can generate a new secret in **System Console > Plugins > Jira**, and paste the new webhook URL in your Jira webhook configuration.
```
Comment (justinegeffen) [inline docs/administrator-guide/frequently-asked-questions-faq.md:43]:
```suggestion
Generate a new secret in **System Console > Plugins > Jira**, then paste the new webhook URL in your Jira webhook configuration.
```

File: docs/administrator-guide/frequently-asked-questions-faq.md [hunk 2]
```diff
@@ -40,39 +40,14 @@ The `<your-mattermost-url>`, `<your-port>`, `<your-secret>`, `<your-team-url>`,
...
 
 ## What changed in the Jira 2.1 Webhook configuration?
 
 In Jira 2.1 there is a modal window for a "Channel Subscription" to Jira issues. This requires a firehose of events to be sent from Jira to Mattermost, and the Jira plugin then "routes" or "drops" the events to particular channels. The Channel Subscription modal \(which you can access by going to a particular channel, then typing `jira /subscribe`\) provides easy access for Mattermost Channel Admins to setup which notifications they want to receive per channel.
```
Comment (justinegeffen) [inline docs/administrator-guide/frequently-asked-questions-faq.md:47]:
```suggestion
In Jira 2.1 there's a modal window for a "Channel Subscription" to Jira issues. This requires a firehose of events to be sent from Jira to Mattermost, and the Jira plugin then "routes" or "drops" the events to particular channels. The Channel Subscription modal \(which you can access by going to a particular channel, then typing `jira /subscribe`\) provides easy access for Mattermost Channel Admins to set up which notifications they want to receive per channel.
```

File: docs/administrator-guide/frequently-asked-questions-faq.md [hunk 3]
```diff
@@ -40,39 +40,14 @@ The `<your-mattermost-url>`, `<your-port>`, `<your-secret>`, `<your-team-url>`,
 
 ## How do I handle credential rotation for the Jira webhook?
 
-You can generate a new secret in **System Console > Plugins > Jira**, and paste the new webhook URL in your Jira webhook configuration. This might result in downtime of the Jira plugin, but it should only be a few minutes at most.
+You can generate a new secret in **System Console > Plugins > Jira**, and paste the new webhook URL in your Jira webhook configuration.
 
 ## What changed in the Jira 2.1 Webhook configuration?
```
Comment (justinegeffen) [inline docs/administrator-guide/frequently-asked-questions-faq.md:45]:
```suggestion
## What changed in the Jira 2.1 webhook configuration?
```

File: docs/administrator-guide/frequently-asked-questions-faq.md [hunk 4]
```diff
@@ -40,39 +40,14 @@ The `<your-mattermost-url>`, `<your-port>`, `<your-secret>`, `<your-team-url>`,
...
-   * Issue Updated, including when an issue is reopened or resolved, or when the assignee is changed. Optionally send notifications for comments, see below.
-
-**Note:** You can send notifications for comments by selecting **Issue Updated**, then adding `&updated_comments=1` to the end of the webhook URL, such as:
-
-```text
-https://community.mattermost.com/plugins/jira/webhook?secret=5JlVk56KPxX629ujeU3MOuxaiwsPzLwh&team=contributors&channel=town-square&updated_comments=1
-```
-
-This sends all comment notifications to a Mattermost channel, including public and private comments, so be cautious of which channel you send these notifications to.
+If your organization's infrastructure is set up such that your Mattermost instance cannot connect to your Jira instance, you will not be able to use the Channel Subscriptions feature. You will instead need to use the [Legacy Webhooks](admininstrator-guide/notification-management.md#legacy-webhooks) feature supported by the Jira plugin.
```
Comment (justinegeffen) [inline docs/administrator-guide/frequently-asked-questions-faq.md:53]:
```suggestion
If your organization's infrastructure is set up in such a way that your Mattermost instance can't connect to your Jira instance, you won't be able to use the Channel Subscriptions feature. Instead, you need to use the [Legacy Webhooks](admininstrator-guide/notification-management.md#legacy-webhooks) feature supported by the Jira plugin.
```

**Review by justinegeffen — COMMENTED** [review]
Some additional comments that were left out of my first review.

Nothing major - just suggestions to improve the flow. Otherwise LGTM!

**Pull request merged**